storage/integration_test.go

```diff
@@ -2928,31 +2943,23 @@ func TestIntegration_RequesterPays(t *testing.T) {
 			checkforErrors("default object acl delete", bucket.DefaultObjectACL().Delete(ctx, entity))
 
 			// Object ACL operations
-			checkforErrors("object acl set", bucket.Object(objectName2).ACL().Set(ctx, entity, RoleReader))
-			_, err = bucket.Object(objectName2).ACL().List(ctx)
+			checkforErrors("object acl set", bucket.Object(objectName).ACL().Set(ctx, entity, RoleReader))
+			_, err = bucket.Object(objectName).ACL().List(ctx)
 			checkforErrors("object acl list", err)
-			checkforErrors("object acl list", bucket.Object(objectName2).ACL().Delete(ctx, entity))
+			checkforErrors("object acl list", bucket.Object(objectName).ACL().Delete(ctx, entity))
 
 			// Copy and compose
 			_, err = bucket.Object("copy").CopierFrom(bucket.Object(objectName)).Run(ctx)
 			checkforErrors("copy", err)
 			_, err = bucket.Object("compose").ComposerFrom(bucket.Object(objectName), bucket.Object("copy")).Run(ctx)
 			checkforErrors("compose", err)
-
-			// Delete object
-			err = bucket.Object(objectName).Delete(ctx)
```

Comment:
Do we still need to clean up the object in the second bucket in each loop iteration? It looks like a new object has been created each t.Run.

Reply:
yes, check line 2915. Since we are using two objects per loop, I moved the delete test to the object operations section, right before we create the next object to use for the rest of the ops (cleaned up normally on L2929)